packages/react-components/src/models/constants.ts

```diff
@@ -0,0 +1,27 @@
+import { SplunkApplicationNamespace } from './splunk';
+
+export const appName: string = 'flare';
+export const storageRealm: string = 'flare_integration_realm';
+export const applicationNameSpace: SplunkApplicationNamespace = {
+    owner: 'nobody',
+    app: appName,
+    sharing: 'app',
+};
+export const flareSavedSearchName = 'Flare Search';
```

Comment:
Is there a reason why we don't do all upper case for these constants?

Also, I see typing on some of these but then some not. Not sure if it's even that helpful to include typing for scalar constants. like `const appName = 'flare'` can only ever be a string.